python/desc/twinkles/sqlite_tools.py

```diff
@@ -29,3 +31,30 @@ def create(self, columns, table, condition=None):
             query += ' ' + condition
         data = [row for row in self.conn.execute(query)]
         return pd.DataFrame(data=data, columns=columns)
+
```

Comment:
This tool was meant to be a sort of general purpose way of packaging columns from an sqlite file into a data frame.   I'll add access to the `DISTINCT` flag, but more generally, once the user has the data frame, any subsequent selection, including down-selecting rows, can be easily done using the pandas selection facilities.  I guess one could argue that this class is trivial enough that it really is not needed.